benchmarks/profiler/deploy/pvc-access-pod.yaml

```
@@ -0,0 +1,31 @@
+    resources:
+      requests:
+        memory: "128Mi"
+        cpu: "100m"
+      limits:
+        memory: "256Mi"
+        cpu: "200m"
+  volumes:
+  - name: profiling-storage
+    persistentVolumeClaim:
+      claimName: profiling-pvc
+  restartPolicy: Never
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)